db/models/sector.py

```diff
@@ -9,10 +10,11 @@ class Sector(Base):
     __tablename__ = 'sector'
 
     id = Column(String, default=uuid_str, primary_key=True)
-    data = Column(LargeBinary)
+    data = Column(String)
     order = Column(Integer)
-
+    is_used = Column(Boolean, default=False)
```

Comment:
A better way might be creating a property:

`
@property
def is_used(self):
    return data is not None and data != ''
`

Reply:
🙆